src/devices/Ssd1351/Ssd1351.cs

```diff
@@ -0,0 +1,308 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Spi;
+using System.Drawing;
+using System.Threading.Tasks;
+
+using Iot.Device.Ssd1351.Command;
+
+namespace Iot.Device.Ssd1351
+{
+    /// <summary>
+    /// A single-chip CMOS OLED/PLED driver with controller for organic/polymer
+    /// light emitting diode dot-matrix graphic display system. 
+    /// </summary>
+    public partial class Ssd1351 : IDisposable
```

Comment:
Is there a particular reason why you decided to separate the implementation of the driver into two different files?

Reply:
Yes... There was a discussion somewhere on what image functions and imaging libraries should be used as part of a device writing out graphics. I don't think that this has been grounded out but I have seen others use System.Drawing and so I have used it here.

Given this I wanted to separate the code that writes to the device from anything that deals with bitmaps etc and hence the two files.